docs/guides/vuejs.md

```diff
@@ -5,132 +5,27 @@ custom_edit_url: https://github.com/stryker-mutator/stryker-js/edit/master/docs/
-
-### Rebuild unit tests with webpack and mocha
-
-Follow these steps to be able "manually" run webpack and mocha.
-
-1. Install `webpack-cli` and `glob`: `npm i -D webpack-cli glob` or `yarn add --dev webpack-cli glob`.
-2. Create a `webpack.config.stryker.js` file with the following content:
-
-   ```js
-   // webpack.config.stryker.js
-   const glob = require('glob');
-
-   // Set env
-   process.env.BABEL_ENV = 'test';
-   process.env.NODE_ENV = 'test';
-   process.env.VUE_CLI_BABEL_TARGET_NODE = 'true';
-   process.env.VUE_CLI_TRANSPILE_BABEL_RUNTIME = 'true';
-   process.env.BUILD_ENV = 'testing';
-
-   // Load webpack config
-   const conf = require('@vue/cli-service/webpack.config.js');
-
-   // Override the entry files
-   conf.entry = {
-     // Choose your test files here:
-     tests: ['./test/setup-unit.js', ...glob.sync('src/**/*+(spec).js').map((fileName) => `./${fileName}`)],
-   };
-
-   module.exports = conf;
-   ```
-
-3. Try it out: `npx webpack --config webpack.config.stryker.js`. This should create a `dist` directory with these files `dist/js/chunk-vendors.js` and `dist/js/tests.js`.
-
-   ```
-   $ npx webpack --config webpack.config.stryker.js
-   Starting type checking service...
-   Using 1 worker with 2048MB memory limit
-
-    DONE  Compiled successfully in 5343ms
-
-   Hash: 8a4d024467cd0b96397e
-   Version: webpack 4.44.2
-   Time: 5343ms
-   Built at: 10/06/2020 9:49:20 PM
-               Asset      Size  Chunks             Chunk Names
-         favicon.ico  4.19 KiB          [emitted]
-          index.html  1.02 KiB          [emitted]
-         js/tests.js  1000 KiB   tests  [emitted]  tests
-        version.json  91 bytes          [emitted]
-   Entrypoint tests = js/tests.js
-   ```
-
-4. Now test out a test run with `mocha`. Run `npx mocha --require @vue/cli-plugin-unit-mocha/setup.js dist/js/chunk-vendors.js dist/js/tests.js`.
-
-This needs to work first before progressing to the next step. You might need to tweak your `webpack.config.stryker.js` file in order to make this work.
-
-### Configure and run Stryker
-
-Once mocha runs successfully on the webpack output, you're ready to install and run Stryker:
-
-1. Install the `@stryker-mutator/mocha-runner` plugin: `npm i -D @stryker-mutator/mocha-runner` or `yarn add --dev @stryker-mutator/mocha-runner`.
-2. Create your `stryker.conf.json` file:
-   ```json
-   {
-     "$schema": "./node_modules/@stryker-mutator/core/schema/stryker-schema.json",
-     "testRunner": "mocha",
-     "concurrency": 2,
+     "$schema": "../../node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+     "testRunner": "vitest",
+     "concurrency": 1,
```

Comment:
Why concurrency 1?

Reply:
Because i copied the config from the e2e test